INSTALL.md

```diff
@@ -0,0 +1,78 @@
+#
+# Copyright (C) 2013 Canonical Ltd
+# Copyright (C) 2017-2021, Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+# Build and installation instructions
+
+## Compile-time build dependencies
+
+ - gettext (>= 0.18.1.1-10ubuntu3)
+ - glib2.0 (>= 2.35.4)
+ - libayatana-common-dev (>= 0.9.1)
+ - cmake (>= 3.13)
+ - gcovr (>= 2.4)
+ - lcov (>= 1.9)
+ - googletest (>= 1.6.0)
```

Comment:
Since you agreed:

glib2.0 -> glib-2.0
libayatana-common-dev -> libayatana-common
googletest -> gtest